docs/release-notes/v11.5.0.md

```diff
@@ -0,0 +1,115 @@
+---
+id: v11.5.0
+title: Version v11.5.0 Release Notes
+sidebar_label: v11.5.0
+image: "img/release-note-link-preview.png"
+---
+
+_Aug 31st, 2023_
+
+## Console
+
+### New Features
+
+#### Project Settings 
+
+Inside the 'Overview' area of a Project, you can now access a brand new section called 'Settings', where users with enough administrative permissions can view and manage the settings of their Project. This will speed-up and optimize the overall administration of Projects, which – until now – was only possible exclusively via CMS, making it no longer tedious or error-prone!
+
+![Project Settings](./img/overview/project-settings.png)
+
+For more information about Project Settings, [go to the official documentation](/development_suite/set-up-infrastructure/project-settings.md).
+
+### Improvements
+
+#### User Profile section 
+
+We have enhanced the viewing and management of user profile settings and preferences, combining them into a single section always accessible from the Console launchbar. As a matter of fact, by clicking on the new `Profile` option from the dropdown menu, you will be presented with a section containing two tabs: `Personal info` and `Preferences`.  
+In the `Personal info` tab, you will find a brief summary of your profile and an external link for managing your profile settings on the identity provider.  
+Within the "Preferences" tab, you can view and manage user's specific preferences.
+
+### Bug Fix
+
+This version addressed the following bugs:
+
+* When configuring a _Single View Creator Service_ inside the `Single Views` section, creating a new _Validator_ will generate a file with an empty function to avoid to automatically save an empty file that might cause a crash of the Single View Creator Service configured.
```

Comment:
wrong place?

Reply:
Yes, my bad.

I've just removed this part and left it as originally created.